gulp-tasks/nodemon.js

```diff
@@ -1,7 +1,18 @@
 var gulp = require('gulp');
 var nodemon = require('gulp-nodemon');
+var env = require('gulp-env');
 
 gulp.task('nodemon', function() {
+  const PROD_PORT = 8080;
+  const DEV_PORT = 8081;
+  // This results in different ports for prod and dev
+  var port = (GLOBAL.config.env === 'prod') ? PROD_PORT : DEV_PORT;
```

Comment:
I wasn't very clear in https://github.com/GoogleChrome/application-shell/issues/64#issuecomment-165791488

Some hosting providers (at least Heroku and App Engine, though it's probably more than those) require that your Node.js server listens on whatever **they** set `process.env.PORT` to. I think they want the flexibility to start up your process on whatever port they choose, based on whatever restrictions they have in place.

So what I'd suggest is to always use `process.env.PORT` if it's already set, and if it's not set, then use one port number for 'prod' and a separate port number for 'dev'.

Maybe something like `var port = process.env.PORT || ((GLOBAL.config.env === 'prod') ? PROD_PORT : DEV_PORT);`